spec/controllers/ideas_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe IdeasController, type: :controller do
+  before(:each) do
+    @idea = Idea.create(title: "Title Sample", body: "Body Sample")
+  end 
+  it "index" do 
+    get :index, format: :json
+    expect(response.status).to eq(200)
+    ideas = JSON.parse(response.body)
+    idea = ideas.first
+    expect(idea['title']).to eq('Title Sample')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.